internal/app/packages.json

```diff
@@ -16674,5 +16674,32 @@
         "liquibaseCore": "4.11.0"
       }
     ]
+  },
+  {
+    "name": "liquibase-checks",
+    "category": "pro",
+    "versions": [
+      {
+        "tag": "1.0.0",
```

Comment:
@mcred : would it cause an issue to have same tag and checksum for different "liquibaseCore" version? I dont think it should but just double checking.